authentication-service/src/main/kotlin/com/saveourtool/save/authservice/security/ConvertingAuthenticationManager.kt

```diff
@@ -62,4 +63,8 @@ class ConvertingAuthenticationManager(
                     identitySource = identitySource,
                 )
             }
+
+    companion object Factory {
+        const val AUTH_SEPARATOR = "@SAVE@"
```

Comment:
I think all constants should be in our Constants.kt

Reply:
Fixed